app/tests/unit/routes/v1/admin.spec.js

```diff
@@ -211,13 +211,13 @@ describe(`GET ${basePath}/users/userId`, () => {
 });
 
 
-describe(`GET ${basePath}/formusers`, () => {
+describe(`GET ${basePath}/forms/formId/formusers`, () => {
```

Comment:
formUsers the U should be capitalized. Check other files and locations for case consistency.

Reply:
Fixed. Searched code